cmd/loopix-client/run.go

```diff
@@ -78,11 +62,28 @@ func cmdRun(args []string, usage string) {
 	var pubC *sphinx.PublicKey
 	var demoRecipient config.ClientConfig
 
+	// nasty hack to make demo work
```

Comment:
Is this still needed now that we're out of demo-land and into using the directory server?

Reply:
Once we're out of demo-land it won't be necessary, it's basically part of hardcoding our recipient.